### PR TITLE
RENO-1827: Adds Tests for LocationHours Component

### PR DESCRIPTION
**Pull request by nypl-wluisi**
[Jira Ticket](http://jira.nypl.org/browse/RENO-1827)

**This PR does the following:**
- Add test coverage for `LocationHours` component.

### Review Steps:
- [ ] Pull in branch, run `npm install`
- [ ] Run `npm test` -- all tests should pass.

### CODE REVIEW

File: src/components/location-finder/Location/LocationHours.test.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        todayHoursStart={'10:08'}
+        todayHoursEnd={'17:57'}
+      />
+    );
+
+    expect(screen.getByText("Today's Hours:")).toBeInTheDocument();
+    expect(screen.getByText('10:08AM–5:57PM')).toBeInTheDocument();
+  });
+
+  test('location shoule be closed due to regular hours', () => {
```
Comment (EdwinGuzman) [inline src/components/location-finder/Location/LocationHours.test.js:63]: Small typo.

File: src/components/location-finder/Location/LocationHours.test.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    render(
+      <LocationHours
+        open={true}
+        todayHoursStart={null}
+        todayHoursEnd={null}
+      />
+    );
+
+    expect(screen.getByText('Closed.')).toBeInTheDocument();
```
Comment (EdwinGuzman) [inline src/components/location-finder/Location/LocationHours.test.js:72]: Does this also render "Today's Hours:"?

**Review by EdwinGuzman — COMMENTED** [review]
Looks good and covered a lot of cases. What happens if one of the hours is `null`?

File: src/components/location-finder/Location/LocationHours.test.js (hunk 1 above)
Comment (nypl-wluisi) [reply src/components/location-finder/Location/LocationHours.test.js]: This should be fixed now.

File: src/components/location-finder/Location/LocationHours.test.js (hunk 2 above)
Comment (nypl-wluisi) [reply src/components/location-finder/Location/LocationHours.test.js:72]: yeah it would render "Today's Hours: Closed"

**Comment by nypl-wluisi** [comment]
I fixed the typo and added 2 tests for start hours null only and end hours only null.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**